Jenkinsfile

```diff
@@ -982,16 +1080,40 @@ pipeline {
                         label 'ci_vm9'
                     }
                     steps {
-                        functionalTest inst_repos: el7_daos_repos(),
-                                       inst_rpms: get_daos_packages('centos7') +
-                                                  ' ' + functional_rpms
+                        functionalTest inst_repos: daos_repos("centos7"),
+                                       inst_rpms: get_daos_packages('centos7') + ' ' +
+                                                  functional_rpms('centos7') + ' ' +
+                                                  qb_inst_rpms_functional()
                     }
                     post {
                         always {
                             functionalTestPost()
                         }
                     }
                 }
+                stage('Functional on Leap 15') {
+                    when {
+                        beforeAgent true
+                        expression { ! skip_stage('func-test-leap15') }
+                    }
+                    agent {
+                        label 'ci_vm9'
+                    }
+                    steps {
+                        functionalTest target: 'leap15',
+                                       inst_repos: daos_repos('leap15'),
+                                       inst_rpms: get_daos_packages('leap15') + ' ' +
+                                                  functional_rpms('leap15') + ' ' +
+                                                  qb_inst_rpms_functional()
+                    }
+                    post {
+                        always {
+                            functional_post_always()
+                            archiveArtifacts artifacts: 'Functional/**'
+                            junit 'Functional/*/results.xml, install/lib/daos/TESTING/ftest/*_results.xml'
```

Comment:
I think that functionalTestPost() should cover these steps.

Reply:
Ahhh.  Right.  This new stage didn't catch up with the recent refactoring  Will fix...